types/module/server/manager.go

```diff
@@ -80,7 +80,7 @@ func (mm *Manager) RegisterGRPCGatewayRoutes(apiSvr *api.Server) {
 
 // RegisterModules registers modules with the Manager and registers their services.
 func (mm *Manager) RegisterModules(modules []module.Module) error {
-	mm.modules = modules
+	mm.modules = append(mm.modules, modules...)
```

Comment:
we can avoid this by moving ecocredit module wiring into stable config.

Reply:
update, PR here: https://github.com/regen-network/regen-ledger/pull/524

Reply:
ah, that would duplicate the integration code for ecocredit module in stable and experimental modules. Let's keep it as-is.